README.md

```diff
@@ -20,59 +20,60 @@
   </a>
 </p>
 
+*Spring Cloud Data Flow* is a microservices-based streaming and batch data processing pipelines in Cloud Foundry and
```

Comment:
With the removal of `toolkit`, there is some disconnect here.

Reply:
toolkit for building